src/components/bootstrap/bootstrap.scss

```diff
@@ -1,8 +1,14 @@
-@import url('~bootstrap/dist/css/bootstrap.css');
 
 $white: rgb(255, 255, 255);
 $gold: rgb(246, 205, 77);
 $darkgrey: rgb(27, 41, 48);
 
 $facebook-bg-color: rgb(60, 90, 150);
-$twitter-bg-color: rgb(89, 173, 235);
\ No newline at end of file
+$twitter-bg-color: rgb(89, 173, 235);
+
+@import "~bootstrap/scss/bootstrap-reboot";
```

Comment:
I think we should keep `import`s first right?  I think generally the convention is to apply vendor styles first, then app specific styles.